docs/development/building/Building-in-Fedora.md

```
@@ -1,41 +1,98 @@
+
+```
+$ sudo dnf install chromium
+```
+
+Open `http://localhost:8080/` in Chromium to use the configurator's development build with serial device access.
+
+### Serial Permissions
+
+If ModemManager is installed, remove it to prevent it from interfering with flight controller connections:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)